packages/web/src/pages/chat-page/components/ChatListBlastItem.tsx

```diff
@@ -62,7 +66,46 @@ export const ChatListBlastItem = (props: ChatListBlastItemProps) => {
     { disabled: !audienceContentId || audienceContentType !== 'album' }
   )
 
-  const audienceCount = user?.follower_count ?? 0
+  const { data: purchasersCount } = useGetPurchasersCount(
+    {
+      userId: currentUserId!,
+      contentId: decodedContentId,
+      contentType: audienceContentType
+    },
+    {
+      disabled: audience !== ChatBlastAudience.CUSTOMERS || !currentUserId
+    }
+  )
+  const { data: remixersCount } = useGetRemixersCount(
+    {
+      userId: currentUserId!,
+      trackId: decodedContentId
+    },
+    {
+      disabled: audience !== ChatBlastAudience.REMIXERS || !currentUserId
+    }
+  )
+
+  const audienceCount = useMemo(() => {
```

Comment:
wonder if we'll want this as a utility elsewhere

we could take all this new code and move to a custom hook

not blocking for this change, but keep in mind if we see other cases where we need this value